plugins/modules/lvg.py

```diff
@@ -292,6 +297,9 @@ def make_vg(module, vg_name):
     pvs = module.params['pvs']
     opt = build_vg_opts(module)
 
+    if quorum:
+        result['msg'] += "Attribute quorom is not supported while changing volume group %s.\n" % vg_name
+
```

Comment:
Please move this conditional check to the main function to facilitate fast failure and use module.fail_json to return the error message and exit. You should include this check at line no 791 in the main function of original file. Also, you need to initialise the quorum variable before using it.

Reply:
This is resolved.